common/static/common/js/spec/main_requirejs.js

```diff
@@ -153,6 +155,10 @@
         }
     });
 
+    // None of the common source files require jquery-migrate
+    // (it is loaded by LMS via base_vendor_js, and CMS by common_libraries).
+    require(['jquery-migrate']);
```

Comment:
I was imagining that we might eventually need to load other libraries before running tests, similar to the use case of `common_libraries.js`. I'm cool with keeping this for now since it's just one library.